docs/releases/1.9.13.txt

```diff
@@ -6,6 +6,14 @@ Django 1.9.13 release notes
 
 Django 1.9.13 fixes a bug in 1.9.12.
 
+PosgreSQL Schema Objects
```

Comment:
I added that line.  The ticket I reference in the comment is a different one, for which there
is a work-around in the code that I had to update (if that's what you're referring to).